tests/v2/test_waku_noise.nim

```diff
@@ -2,22 +2,115 @@
 
 import
   testutils/unittests,
+  std/random,
+  stew/byteutils,
   ../../waku/v2/protocol/waku_noise/noise,
   ../test_helpers
 
-
 procSuite "Waku Noise":
   
   let rng = rng()
 
-  test "ChaChaPoly Encryption/Decryption":
+  test "ChaChaPoly Encryption/Decryption: random byte sequences":
 
     let cipherState = randomChaChaPolyCipherState(rng[])
 
+    # We encrypt/decrypt random byte sequences
     let
-      plaintext: seq[byte] = randomSeqByte(rng[], 128)
+      plaintext: seq[byte] = randomSeqByte(rng[], 64)
       ciphertext: ChaChaPolyCiphertext = encrypt(cipherState, plaintext)
       decryptedCiphertext: seq[byte] = decrypt(cipherState, ciphertext)
 
     check: 
-      plaintext == decryptedCiphertext
\ No newline at end of file
+      plaintext == decryptedCiphertext
+
+  test "ChaChaPoly Encryption/Decryption: random strings":
+
+    let cipherState = randomChaChaPolyCipherState(rng[])
+
+    # We encrypt/decrypt random strings
+    randomize()
+    var plaintext: string
+    for _ in .. rand(64):
+      add(plaintext, char(rand(int('A') .. int('z'))))
+
+    echo plaintext
```

Comment:
if it is necessary to check the plaintext, then please use `debug` like https://github.com/status-im/nim-waku/blob/5a77d6e2a641616d62c778da2d8937ecb25644d7/tests/v2/test_waku_rln_relay.nim#L100

More on this subject https://github.com/status-im/nim-chronicles

Reply:
Thanks, used while testing. Forgot to remove.